composer.json

```diff
@@ -22,7 +22,6 @@
         "cweagans/composer-patches": "^1.6.5",
         "drupal-composer/drupal-scaffold": "^2.5",
         "drupal/console": "^1.0.2",
-        "drupal/core": "8.6.x-dev",
```

Comment:
i'll argue to keep this line (albeit the modified version from https://github.com/govCMS/govcms8lagoon/pull/59) in govcms8lagoon for the time being - at least until we've worked out how to advance test the next Drupal release.

Reply:
Okay :(